operators/understanding/olm/olm-operatorconditions.adoc

```diff
@@ -17,3 +17,5 @@ include::modules/olm-supported-operatorconditions.adoc[leveloffset=+1]
 
 * xref:../../../operators/admin/olm-managing-operatorconditions.adoc#olm-operatorconditions[Managing Operator conditions]
 * xref:../../../operators/operator_sdk/osdk-generating-csvs.adoc#osdk-operatorconditions_osdk-generating-csvs[Enabling Operator conditions]
+* xref:../../../nodes/pods/nodes-pods-configuring.adoc#nodes-pods-configuring-pod-distruption-about_nodes-pods-configuring[Understanding how to use pod disruption budgets to specify the number of pods that must be up]
```

Comment:
[Out of scope, food for thought] Is there a reason not to call this topic just "Using pod disruption budgets to specify the number of pods that must be up" to get it a leeeetle shorter?